static/public/images/badges/pp_web.svg

```diff
@@ -0,0 +1 @@
3 1.28-.568l2.156 4.947c-.4.174-.8.352-1.195.532zm59.624-1.036c-.398-.166-.8-.328-1.203-.486l1.98-5.022c.434.17.864.346 1.29.52l-2.066 4.99zm-57.22-.008l-2.073-4.987c.434-.18.868-.355 1.305-.527l1.98 5.023c-.408.16-.81.32-1.212.49zm54.8-.95c-.41-.15-.817-.302-1.23-.446l1.808-5.088c.438.156.877.314 1.313.48l-1.892 5.055zm-52.37-.01l-1.89-5.057c.438-.164.88-.324 1.32-.48l1.8 5.09c-.41.146-.82.294-1.23.447zm49.91-.86c-.413-.14-.825-.27-1.24-.402l1.62-5.15c.447.14.892.285 1.334.434l-1.714 5.118zm-47.44-.015l-1.716-5.12c.445-.147.892-.292 1.34-.433l1.62 5.15c-.416.13-.83.266-1.243.403zm44.953-.77c-.416-.123-.835-.242-1.253-.357l1.44-5.203c.448.123.896.252 1.345.385l-1.533 5.175zm-42.46-.01l-1.53-5.18c.446-.132.896-.26 1.345-.384l1.44 5.204c-.42.115-.84.235-1.256.36zm39.947-.685c-.42-.107-.843-.213-1.266-.314l1.256-5.252c.454.11.907.223 1.357.338l-1.346 5.228zm-37.43-.014L65.22 7.93c.45-.117.905-.23 1.36-.338l1.254 5.25c-.422.1-.844.207-1.263.314zm34.893-.594c-.425-.094-.85-.184-1.277-.27L101.265 7c.456.092.91.19 1.367.29l-1.165 5.27zm-32.36-.01L67.944 7.28c.456-.1.912-.197 1.37-.29l1.068 5.292c-.426.085-.85.175-1.275.27zm29.802-.51c-.43-.077-.855-.15-1.286-.224l.887-5.324c.462.076.92.158 1.378.24l-.98 5.31zm-27.248-.007l-.975-5.31c.458-.085.918-.165 1.377-.24l.885 5.325c-.43.073-.86.147-1.288.225zm24.676-.418c-.43-.064-.862-.125-1.295-.182l.7-5.352c.465.06.926.125 1.386.19l-.792 5.344zm-22.1-.008l-.79-5.343c.46-.068.92-.13 1.384-.19l.698 5.353c-.43.056-.862.115-1.293.18zm19.51-.333c-.43-.047-.862-.094-1.297-.135l.514-5.373c.465.042.928.09 1.392.144l-.607 5.363zm-16.924-.004l-.603-5.365c.465-.053.928-.102 1.393-.145l.51 5.375c-.435.042-.868.087-1.3.135zm14.328-.244c-.433-.032-.867-.063-1.302-.09l.328-5.39c.465.03.93.062 1.394.098l-.42 5.38zm-11.73-.003l-.415-5.383c.463-.035.928-.068 1.396-.098l.322 5.39c-.436.027-.87.058-1.303.09zm9.125-.154c-.433-.02-.865-.033-1.3-.045l.14-5.396c.465.012.93.03 1.395.047l-.235 5.393zm-6.52-.002l-.23-5.394c.466-.02.93-.035 1.397-.047l.138 5.395c-.435.013-.87.027-1.306.046zm3.916-.065l-.688-.002-.62.002-.04-5.398.668-.004.727.004-.047 5.398zM10.512 85.555h-5.4l.003-.698 5.4.047M142.958 85.55h-5.398c0-28.974-23.573-52.546-52.547-52.546-28.973 0-52.545 23.572-52.545 52.545H27.07c0-31.952 25.993-57.946 57.942-57.946 31.952 0 57.946 25.994 57.946 57.945zM20.32 85.55h-5.398c0-7.6 1.188-15.085 3.528-22.245l5.132 1.678C21.418 71.6 20.32 78.52 20.32 85.55z"/><path fill="#F4991D" d="M29.943 50.537L25.5 47.473c13.368-19.39 35.393-30.967 58.914-30.967 23.267 0 45.157 11.383 58.554 30.443l-4.42 3.104c-12.385-17.627-32.622-28.148-54.134-28.148-21.747 0-42.11 10.7-54.47 28.63zM154.582 85.55h-5.398c0-7.33-1.19-14.526-3.538-21.394l5.108-1.746c2.54 7.43 3.828 15.215 3.828 23.14z"/><path fill="#FFD823" d="M.248 93.322H170.85v4.5H.248zM10.473 117.418h150.152v4.5H10.473zM3.437 105.37H167.66v4.5H3.438zM19.777 129.467H151.32v4.5H19.777zM41.515 153.562h88.068v4.5H41.515zM29.81 141.514h111.48v4.5H29.81zM65.743 165.61h41.612v4.5H65.743z"/></svg>
```

Comment:
Should really have just one decimal place precision, but since I have no access to editing app and SVGO **slightly** deforms paths at 1 decimal place (probably invisible at the size used), here it is at two decimal places (40% smaller):
`<svg xmlns="http://www.w3.org/2000/svg" width="171.1" height="170.11" viewBox="0 0 171.1 170.11"><path fill="#FFD823" d="M171.1 85.55C171.1 38.3 132.8 0 85.55 0S0 38.3 0 85.55h171.1z"/><path fill="#F4991D" d="M165.4 85.56H160v-.65l5.4-.05v.7zM160 83.6l-.05-1.3 5.4-.24c0 .46.02.93.04 1.4l-5.4.14zm-149.46-.06l-5.4-.15.05-1.4 5.4.24c-.03.43-.05.86-.06 1.3zM159.88 81l-.1-1.3 5.4-.43.1 1.4-5.4.32zm-149.23-.06l-5.4-.33.1-1.4 5.4.44-.1 1.3zm149.03-2.55l-.14-1.32 5.37-.6.15 1.4-5.37.5zm-148.83-.07l-5.37-.5c.04-.48.1-.94.15-1.4l5.36.6-.15 1.3zm148.53-2.54c-.05-.44-.1-.87-.18-1.3l5.34-.8.2 1.4-5.36.7zm-148.23-.06l-5.36-.7c.05-.46.12-.93.2-1.4l5.33.8-.18 1.3zM159 73.22l-.22-1.3 5.3-.97c.1.46.17.92.25 1.38l-5.33.9zm-147.47-.06l-5.32-.9c.1-.45.17-.9.25-1.36l5.3.98-.22 1.28zm147-2.5l-.27-1.3 5.27-1.15.3 1.38-5.3 1.07zM12 70.6l-5.28-1.08.3-1.37 5.26 1.17c-.1.42-.2.85-.27 1.28zm145.97-2.5c-.1-.42-.2-.84-.32-1.26l5.23-1.35.34 1.35-5.25 1.25zm-145.4-.05L7.32 66.8c.1-.46.22-.92.34-1.37l5.23 1.35c-.12.42-.23.84-.33 1.27zm144.74-2.48c-.1-.42-.22-.83-.34-1.25l5.17-1.53.4 1.34-5.22 1.43zm-144.08-.05l-5.2-1.44.4-1.34 5.16 1.53-.36 1.26zm143.35-2.45l-.4-1.24 5.12-1.72.42 1.35-5.15 1.62zm-142.6-.05L8.8 61.4c.14-.45.28-.9.43-1.33l5.12 1.7-.4 1.25zm141.77-2.43c-.14-.42-.3-.83-.45-1.24l5.05-1.9c.16.44.33.88.48 1.33l-5.1 1.8zM14.8 60.54l-5.1-1.8.48-1.32 5.06 1.9-.45 1.22zm140.02-2.4c-.16-.42-.32-.82-.5-1.23l5-2.07.53 1.3-5.02 2zm-139.1-.04l-5.04-1.98.53-1.3 5 2.08-.5 1.2zm138.1-2.38l-.53-1.2 4.9-2.24.57 1.28-4.95 2.16zm-137.12 0l-4.95-2.17.57-1.27 4.9 2.24-.52 1.2zm1.08-2.38l-4.87-2.32.62-1.26 4.83 2.42c-.2.38-.38.77-.57 1.16zm134.95 0c-.18-.4-.38-.78-.57-1.17l4.83-2.42.6 1.26-4.86 2.34zm-133.8-2.32l-4.77-2.5.66-1.23 4.74 2.57-.62 1.15zM151.57 51l-.6-1.15 4.73-2.58c.2.4.43.82.65 1.23l-4.8 2.5zM20.2 48.73l-4.7-2.66c.22-.4.46-.8.7-1.2l4.64 2.74-.65 1.13zm130.1-.03l-.64-1.12 4.64-2.75c.24.4.48.8.7 1.2l-4.7 2.67zM21.5 46.5l-4.6-2.84.75-1.18 4.55 2.9-.7 1.1zm127.5-.04l-.7-1.1 4.54-2.92.74 1.2-4.6 2.82zM22.9 44.3l-4.5-3 .78-1.14 4.45 3.06c-.25.35-.5.7-.73 1.08zm124.66-.04c-.24-.37-.48-.73-.73-1.08l4.44-3.07.78 1.17-4.5 3zm-123.2-2.1L20 39l.82-1.12 4.34 3.22c-.26.34-.5.7-.77 1.05zm121.72-.06c-.26-.35-.5-.7-.77-1.05l4.33-3.22.83 1.13-4.4 3.14zM25.94 40.06l-4.28-3.3c.3-.36.57-.73.86-1.1l4.23 3.37-.8 1.03zm118.6-.05l-.8-1.02 4.2-3.36c.3.36.6.73.88 1.1l-4.28 3.3zM27.57 38.03l-4.16-3.44.9-1.07 4.1 3.52-.83 1zm115.34-.06c-.26-.33-.55-.66-.83-1l4.1-3.5.9 1.07-4.16 3.44zM29.27 36.05l-4.04-3.58c.3-.35.62-.7.94-1.04l3.98 3.65-.88.96zM141.22 36c-.3-.34-.6-.66-.88-1l3.98-3.63.94 1.03-4.04 3.58zm-110.2-1.9l-3.9-3.7c.3-.34.63-.68.96-1l3.85 3.77-.9.94zm108.43-.04l-.9-.94 3.84-3.8.96 1.02-3.9 3.72zm-106.6-1.8l-3.78-3.86 1-.97 3.72 3.9c-.32.3-.64.6-.95.92zm104.77-.06l-.94-.9 3.72-3.92 1 .97-3.78 3.85zM34.75 30.45l-3.65-3.98 1.04-.93 3.58 4.04-.97.87zm100.98-.04c-.32-.3-.65-.58-.97-.86l3.57-4.05c.35.3.7.6 1.04.92l-3.64 4zM36.7 28.73l-3.5-4.1 1.06-.9 3.44 4.16-1 .84zm97.07-.04l-1-.84 3.44-4.17 1.08.9-3.5 4.1zM38.7 27.06l-3.35-4.22c.36-.3.73-.58 1.1-.86l3.3 4.28c-.36.26-.7.53-1.04.8zm93.06-.04l-1.02-.8 3.28-4.28 1.1.86-3.36 4.22zm-90.98-1.55l-3.2-4.34c.36-.28.74-.55 1.12-.82l3.14 4.4-1.06.77zm88.92-.04l-1.06-.76 3.14-4.4 1.13.82-3.2 4.33zm-86.8-1.48l-3.05-4.45 1.16-.78 3 4.5-1.1.73zm84.68-.03c-.36-.24-.72-.5-1.08-.73l2.98-4.5c.4.25.77.5 1.15.77l-3.05 4.45zm-82.5-1.4l-2.9-4.56 1.2-.75 2.8 4.6-1.1.7zm80.32-.04l-1.1-.7 2.82-4.6 1.2.75-2.9 4.55zm-78.1-1.34l-2.73-4.65c.4-.25.8-.48 1.2-.7l2.67 4.7c-.38.2-.75.42-1.13.64zm75.9-.02l-1.14-.65 2.66-4.7 1.2.7-2.73 4.65zm-73.6-1.27L47 15.1c.42-.2.83-.43 1.24-.65l2.5 4.8-1.15.6zm71.32 0c-.38-.22-.76-.42-1.15-.63l2.5-4.78 1.23.65-2.58 4.74zm-69-1.2l-2.43-4.84 1.25-.6 2.33 4.87-1.17.57zm66.7-.02l-1.17-.56 2.32-4.88 1.26.6-2.42 4.83zm-2.35-1.12l-1.2-.5 2.17-4.96 1.27.56-2.23 4.9zm-62 0L52 12.6c.42-.2.85-.38 1.28-.57l2.16 4.95-1.2.53zm59.62-1.02l-1.22-.5 1.98-5 1.3.5-2.07 5zm-57.23 0l-2.07-5 1.3-.52 1.98 5.02c-.4.16-.8.32-1.2.5zm54.8-.96l-1.23-.45 1.8-5.08c.44.14.88.3 1.32.47l-1.9 5.05zm-52.37 0l-1.9-5.07 1.33-.48 1.8 5.1-1.23.44zm49.9-.87l-1.23-.4 1.62-5.15 1.32.43-1.7 5.12zm-47.43-.02L59.85 9.5c.45-.13.9-.28 1.34-.42l1.6 5.15-1.23.4zm44.95-.77l-1.25-.35 1.44-5.2 1.35.4-1.53 5.16zm-42.46 0l-1.53-5.2 1.35-.37 1.44 5.2c-.42.1-.84.23-1.26.35zm39.95-.7l-1.26-.3L104 7.6c.45.1.9.23 1.36.34L104 13.17zm-37.42 0l-1.36-5.23 1.36-.34 1.25 5.24c-.42.1-.84.2-1.26.32zm34.9-.6c-.43-.1-.86-.18-1.3-.27l1.08-5.3 1.37.3-1.16 5.26zm-32.37 0l-1.16-5.28 1.37-.3 1.08 5.3c-.42.1-.85.18-1.27.27zm29.8-.52l-1.27-.22.88-5.33 1.4.23-1 5.3zm-27.23 0l-.98-5.32 1.37-.24.88 5.33-1.3.23zm24.67-.42l-1.3-.2.7-5.34 1.4.2-.8 5.34zm-22.1 0l-.8-5.35 1.4-.2.7 5.36c-.44.05-.87.1-1.3.18zm19.5-.34l-1.3-.14.53-5.37 1.4.14-.62 5.37zm-16.9 0L76.2 5.9l1.4-.14.5 5.38-1.3.13zm14.32-.25l-1.3-.1.32-5.38 1.4.1-.42 5.38zm-11.73 0L79 5.62c.47-.02.94-.06 1.4-.1l.33 5.4-1.3.1zm9.12-.16l-1.3-.04.14-5.4c.46 0 .92.03 1.4.05l-.25 5.4zm-6.52 0l-.23-5.4 1.4-.04.14 5.4c-.44 0-.87.02-1.3.04zm3.92-.07H84.65l-.05-5.4H86l-.05 5.4zM10.5 85.55H5.1l.02-.7 5.4.05m132.44.65h-5.4C137.56 56.58 114 33 85 33c-28.96 0-52.53 23.58-52.53 52.55h-5.4c0-31.95 26-57.95 57.94-57.95 31.96 0 57.96 26 57.96 57.95zm-122.64 0h-5.4c0-7.6 1.2-15.1 3.53-22.25l5.13 1.68c-2.16 6.62-3.26 13.54-3.26 20.57z"/><path fill="#F4991D" d="M29.94 50.54l-4.44-3.07C38.87 28.07 60.9 16.5 84.4 16.5c23.28 0 45.17 11.4 58.57 30.45l-4.42 3.1C126.15 32.43 105.93 21.9 84.4 21.9c-21.73 0-42.1 10.7-54.46 28.63zm124.64 35h-5.4c0-7.32-1.2-14.52-3.53-21.38l5.1-1.75c2.54 7.44 3.83 15.23 3.83 23.15z"/><path fill="#FFD823" d="M.25 93.32h170.6v4.5H.25zm10.22 24.1h150.15v4.5H10.47zm-7.03-12.05h164.22v4.5H3.44zm16.34 24.1h131.54v4.5H19.78zm21.73 24.1h88.08v4.5H41.5zm-11.7-12.06h111.5v4.5H29.8zm35.94 24.1h41.62v4.5H65.73z"/></svg>`